### DISCUSSION TITLE
Why was walls disabled in bot matches?

### DISCUSSION THREAD

**Discussion by CyberFoxHax** [OpenHV/OpenHV#1230] [Support] [answered]
https://github.com/OpenHV/OpenHV/releases/tag/20240401

at the end it says, "Disabled gates/walls on bot matches. 🤖" with no further information.

**Comment by SiegeSpud** [comment, marked as answer]
AI does not understand walls and it was to easy to cheese it. There is no fix in the forseeable future. Some multiplayer maps have neutral walls but it is mainly for decoration.

**Comment by Mailaender** [comment]
To avoid stalemate situations like https://youtu.be/SCgmMs0_Itw just as @blizzard did in Warcraft 2.

**Reply by CyberFoxHax** [reply]
yeah that does look broken.
